sfaira/_settings.py

```diff
@@ -0,0 +1,72 @@
+"""
+Settings class which for example holds paths to cache directories used throughout the code.
+"""
+
+import os
+
+
+class SfairaConfig:
+    """\
+    Config manager for sfaira.
+    """
+
+    def __init__(self):
+        self.sfaira_repo_url = "https://zenodo.org/record/4836517/files/"
+        self._cachedir_base = os.path.join(os.path.expanduser("~"), ".cache", "sfaira")
+        self._cachedir_databases = os.path.join(self._cachedir_base, "dataset_meta")
+        self._cachedir_databases_cellxgene = os.path.join(self._cachedir_databases, "cellxgene")
+        self._cachedir_genomes = os.path.join(self._cachedir_base, "genomes")
+        self._cachedir_ontologies = os.path.join(self._cachedir_base, "ontologies")
+
+    @property
+    def cachedir_base(self) -> str:
+        os.makedirs(self._cachedir_base, exist_ok=True)
+        return self._cachedir_base
+
+    @cachedir_base.setter
+    def cachedir_base(self, cachedir_base):
+        if not isinstance(cachedir_base, str):
+            raise ValueError(f"cachedir_base needs to be provided as a string, was {type(cachedir_base)}")
```

Comment:
can we add this so that people can easily write into the repo if they want to, using the flag "repo" instead of a path:
```
if cachedir_base == "repo":
   cachedir_base = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), "cache")
```